lib/tty.js

```diff
@@ -72,7 +72,7 @@ ObjectSetPrototypeOf(ReadStream, net.Socket);
 
 ReadStream.prototype.setRawMode = function(flag) {
   flag = !!flag;
-  const err = this._handle.setRawMode(flag);
+  const err = this._handle && this._handle.setRawMode(flag);
```

Comment:
This should be covered by a test. Also, is it correct that this function _does not_ fail if `this._handle == null`?

Reply:
Thank you for the review
I think it's safe as long as this._handle would be null if this function is called from the terminal.
also, I found this check in readline interface:
https://github.com/nodejs/node/blob/f81c62704fc4e3e0eb1b1c813854ad52fbb8fe75/lib/internal/readline/interface.js#L349-L350

But maybe I'm wrong, wdyt ?

Reply:
@tniessen What do you think?

Reply:
```suggestion
  const err = this._handle?.setRawMode(flag);
```